subiquity/core.py

```diff
@@ -46,8 +44,6 @@
 
 class Subiquity(Application):
 
-    signals = ['debug-shell-exited']
```

Comment:
Why is this no longer needed?

Reply:
It was only used to trigger a reprobe when the debug shell exited. If there's other things we might want to trigger then, we can keep it -- but I can't think of anything.